sourcebundle/bundle.go

```diff
@@ -354,6 +362,11 @@ func (b *Bundle) RegistryPackageVersions(pkgAddr regaddr.ModulePackage) versions
 	return ret
 }
 
+func (b *Bundle) RegistryPackageVersionDeprecation(pkgAddr regaddr.ModulePackage, version versions.Version) *RegistryVersionDeprecation {
+	deprecation := b.registryPackageVersionDeprecations[pkgAddr][version]
+	return deprecation
```

Comment:
I thought this could panic, but it wont! Neat. No need for a local, though

```suggestion
	return b.registryPackageVersionDeprecations[pkgAddr][version]
```

Reply:
Included 👍  https://github.com/hashicorp/go-slug/pull/59/commits/ea8f696aaf4615b5eed02231bed0038ff283c397